src/components/Header.vue

```diff
@@ -23,29 +30,52 @@ export default {
           icon: 'supervisor_account',
           title: 'VIEW MEETUPS',
           route: '/meetups',
+          click: () => null,
         },
         {
           icon: 'location_on',
           title: 'ORGANIZE MEETUP',
-          route: '/createmeetups',
+          route: '/createmeetup',
+          click: () => null,
         },
         {
           icon: 'person',
           title: 'PROFILE',
           route: '/profile',
+          click: () => null,
         },
         {
           icon: 'face',
           title: 'SIGN UP',
           route: '/signup',
+          click: () => null,
         },
         {
           icon: 'lock_open',
           title: 'SIGN IN',
           route: '/signin',
+          click: () => null,
+        },
+        {
+          icon: 'login',
+          title: 'LOGOUT',
+          route: this.logOutPath,
+          click: () => this.$store.dispatch('logout'),
```

Comment:
Logout button should not be a link, you need to navigate use programmatically navigation in click event handler. In this case I would suggest to work with it a little bit different than with other toolbar elements, for example using v-if directive.